kind/infra-resources.yaml

```diff
@@ -66,16 +66,28 @@ spec:
         app: kafka-broker
     spec:
       containers:
-        - name: broker
-          image: confluentinc/cp-kafka:7.3.2
+        - name: kafka
+          image: confluentinc/cp-kafka:7.7.1
           imagePullPolicy: IfNotPresent
           env:
+            - name: CLUSTER_ID
+              value: 39926ADE-CE84-48D9-A027-0B937158BC96
```

Comment:
Perhaps someone knows a good way to generate a random uuid for a kind cluster instead of reusing this one?

Reply:
We could mount it using k8s, but I feel like that's not worth it. What's the downside here assuming this cluster will only ever run locally containerized?

Reply:
Yeah, you're probably right. I looked at substituting something from bootstrap.sh, but it felt like overkill.